### DISCUSSION TITLE
Any plans for icons in project view?

### DISCUSSION THREAD

**Discussion by Mr0Bread** [zed-industries/zed#6505] [Feature Requests]
Are there any plans for icons that are displayed right before file name? I mean custom icons for different type of files

**Comment by jbergstroem** [comment]
I believe this is [one of the most requested enhancements](https://github.com/zed-industries/community/issues/52). Issue here: https://github.com/zed-industries/community/issues/206

**Comment by notpeter** [comment]
Zed has had this for a bit.

<img width="334" alt="image" src="https://github.com/user-attachments/assets/458cf8c5-a94b-4571-83e0-cada1b50e8c3">
